Support/Scripts/run-lldb-tests.sh

```diff
@@ -25,12 +25,14 @@ source "$top/Support/Scripts/common.sh"
 [ -x "$build_dir/ds2" ]   || die "Unable to find a ds2 binary in the current directory."
 
 opt_fast=false
+opt_no_ds2_blacklists=false
```

Comment:
Yeah, that sounds reasonable. I wasn't super thrilled with the verbosity of that, but I think you're right about it begin worth it